### PR TITLE
adding simple-pid recipe

### PR DESCRIPTION
**Pull request by kbeeperez**
implements: https://github.com/nsls-ii-forge/simple-pid-feedstock

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kbeeperez** [comment]
Attn @mrakitin

**Comment by kbeeperez** [comment]
Attn @mrakitin

File: recipes/simple-pid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set name = "simple-pid" %}
+{% set version = "1.0.1" %}
+package:
```
Comment (mrakitin) [inline recipes/simple-pid/meta.yaml:3]:
```suggestion

package:
```

File: recipes/simple-pid/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - openpyxl >=3
+    - pandas
+    - py4xs
```
Comment (mrakitin) [inline recipes/simple-pid/meta.yaml:24]: Why do we need beamline-specific packages here? It's a general-purpose package.

Here are the original dependencies: https://github.com/nsls-ii-forge/simple-pid-feedstock/blob/ab6a12b3d6bd0898c23b781ae00945dd9b76e5ce/recipe/meta.yaml#L18-L23.

**Review by mrakitin — CHANGES_REQUESTED** [review]
The requirements should be revised.

@conda-forge/staged-recipes, I agree to maintain this feedstock/package/recipe.

File: recipes/simple-pid/meta.yaml (hunk 2 above)
Comment (kbeeperez) [reply recipes/simple-pid/meta.yaml:25]: In the commands: section of the meta.yaml there are the import lines. It was giving a " no module name: "pkg_resourse" " error and so I compared it to [lixtools](https://github.com/conda-forge/staged-recipes/pull/15648/files) meta.yaml which uses the same type of import command and just put tested different packages from it till it passed all the checks. 

I put in openpyxl, pandas, and py4xs together so I'm not sure which one made it work.

**Comment by mrakitin** [comment]
@kbeeperez, if you have multiple suggestions to merge, I'd recommend using "batch merging". You can go to the Files tab, and add suggestions to a batch, and then commit in one shot. This will save CI runs.

See https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request for more details.

File: recipes/simple-pid/meta.yaml (hunk 2 above)
Comment (mrakitin) [reply recipes/simple-pid/meta.yaml:25]: See https://github.com/nsls-ii-forge/pyfiglet-feedstock/blob/d92d0710d3dfdd97120b1c0c12a061f8c341b33d/recipe/meta.yaml#L26-L27 as an example. You'll need to add `setuptools` to the run dependencies to provide `pkg_resources` lib.

Comment (mrakitin) [reply recipes/simple-pid/meta.yaml:25]: Since it's mostly related to tests, let's add it to the `test` block's dependencies instead.

**Review by mrakitin — APPROVED** [review]
Looks good to me now :)

**Review by dopplershift — APPROVED** [review]

**Pull request merged**